nextmove/src/main/java/no/difi/meldingsutveksling/nextmove/AbstractEntity.java

```diff
@@ -0,0 +1,34 @@
+package no.difi.meldingsutveksling.nextmove;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Getter;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+import javax.xml.bind.annotation.XmlTransient;
+import java.io.Serializable;
+
+@MappedSuperclass
+public abstract class AbstractEntity<I extends Serializable> implements Serializable {
+
+    @Id
+    @GeneratedValue
+    @JsonIgnore
+    @XmlTransient
+    @Getter
+    private I id;
+
+    @Override
+    public final boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS00122)